### DISCUSSION TITLE
Sensor Start in the System Status panel

### DISCUSSION THREAD

**Discussion by vpv1952** [NightscoutFoundation/xDrip#2977] [General]
I've just initialized a new sensor libre 2 (EU) using the abbott reader (that came with libre 1).  after that, I NFC scanned it, with xDrip+ for direct connection.  and finally, I used xdrip start sensor function and everything started working normally.

now, looking at the System Status panel, it shows that the start date is yesterday and indicates "(1d0h).   I'm using the last nightly version.

is this correct?  why does it not show today's date?

**Comment by ANDREAPIPPI** [comment]
Hi. 
Same problem, with nightly 20230716.

**Reply by vpv1952** [reply]
I think that the correct information - the sensor age - is the one displayed in the xdrip home screen.  

the info presented in the classic status page is something else..
